Dockerfile

```diff
@@ -0,0 +1,62 @@
+# build env
+FROM node:16.14.0 as build
+# set working directory
+WORKDIR /app
+# install app dependencies
+COPY package.json ./
+RUN yarn install
+# add app
+COPY . ./
+# install dependencies
+RUN apt-get update && apt-get install -y \
+    gconf-service \
+    libasound2 \
+    libatk1.0-0 \
+    libatk-bridge2.0-0 \
+    libc6 \
+    libcairo2 \
+    libcups2 \
+    libdbus-1-3 \
+    libexpat1 \
+    libfontconfig1 \
+    libgcc1 \
+    libgconf-2-4 \
+    libgdk-pixbuf2.0-0 \
+    libglib2.0-0 \
+    libgtk-3-0 \
+    libnspr4 \
+    libpango-1.0-0 \
+    libpangocairo-1.0-0 \
+    libstdc++6 \
+    libx11-6 \
+    libx11-xcb1 \
+    libxcb1 \
+    libxcomposite1 \
+    libxcursor1 \
+    libxdamage1 \
+    libxext6 \
+    libxfixes3 \
+    libxi6 \
+    libxrandr2 \
+    libxrender1 \
+    libxss1 \
+    libxtst6 \
+    ca-certificates \
+    fonts-liberation \
+    libappindicator1 \
+    libnss3 \
+    lsb-release \
+    xdg-utils \
+    wget \
+    libgbm1
```

Comment:
Why are all this lib* dependencies necessary?

Reply:
Puppeteer needs them to work. [This is the full list of dependencies](https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#:~:text=ca%2Dcertificates%0Afonts,wget%0Axdg%2Dutils)

Reply:
After `yarn build`, the postbuild process runs Puppeteer.  This is the current set behaviour after `yarn build` is called.

Reply:
Oh to pre-render right :+1: